lib/dpl/provider/modulus.rb

```diff
@@ -16,7 +16,8 @@ def needs_key?
       end
 
       def push_app
-        context.shell "MODULUS_TOKEN=#{option(:api_key)} modulus deploy -p #{option(:project_name)}"
+        ENV['MODULUS_TOKEN']="#{option(:api_key)}"
```

Comment:
Does `ENV` affect how commands are executed in `context.shell`? I tend to think it does not. Has this been tested?

Perhaps we can do with `context.shell "env MODULUS_TOKEN=#{option(:api_key)} modulus …"`.